src/sql/platform/telemetry/common/telemetryKeys.ts

```diff
@@ -52,6 +52,7 @@ export const enum TelemetryError {
 }
 
 export const enum TelemetryAction {
+	ActualQueryExecutionPlan = 'ActualQueryExecutionPlan',
```

Comment:
I think actual/estimated execution should be boolean property of ExecuteQuery action.

Reply:
Are you wanting me to replace `ActualQueryExecutionPlan` and `EstimatedQueryExecutionPlan` with something more generic like `ExecutionPlan`, and then the boolean property would be used to distinguish between actual vs estimated?

Reply:
That sounds better to me too - although keep in mind that "TelemetryAction" should be an action. So I would expect something like `ViewExecutionPlan` or `SaveExecutionPlan`.